tests/bbsSignature/verify.bbsSignature.spec.ts

```diff
@@ -121,37 +127,42 @@ describe("bbsSignature", () => {
           "jYidhsdqxvAyNXMV4/vNfGM/4AULfSyfvQiwh+dDd4JtnT5xHnwpzMYdLdHzBYwXaGE1k6ln/pwtI4RwQZpl03SCv/mT/3AdK8PB2y43MGdMSeGTyZGfZf+rUrEDEs3lTfmPK54E+JBzd96gnrF2iQ=="
         ),
       };
-      expect(verify(request).verified).toBeFalsy();
+      expect((await verify(request)).verified).toBeFalsy();
     });
 
-    it("should throw error when public key invalid length", () => {
+    it("should throw error when public key invalid length", async () => {
       const request: BbsVerifyRequest = {
         publicKey: new Uint8Array(20),
         messages: [],
         signature: base64Decode(
           "jYidhsdqxvAyNXMV4/vNfGM/4AULfSyfvQiwh+dDd4JtnT5xHnwpzMYdLdHzBYwXaGE1k6ln/pwtI4RwQZpl03SCv/mT/3AdK8PB2y43MGdMSeGTyZGfZf+rUrEDEs3lTfmPK54E+JBzd96gnrF2iQ=="
         ),
       };
-      expect(verify(request).verified).toBeFalsy();
+      expect((await verify(request)).verified).toBeFalsy();
     });
   });
   describe("blsVerify", () => {
-    const blsKeyPair = generateBls12381G2KeyPair();
-    it("should verify valid signature with a single message", () => {
+    let blsKeyPair: BlsKeyPair;
+
+    beforeAll(async () => {
```

Comment:
nitpick: You can consolidate this call and the one above by moving it below to describe("bbsSignature" on line 29 before starting the verify test module.

Reply:
I think we will refactor theses tests more extensively in the near future so I'll keep these suggestions in mind for then